src/electron-main.ts

```diff
@@ -321,6 +327,19 @@ if (global.store.get("disableHardwareAcceleration", false) === true) {
     app.disableHardwareAcceleration();
 }
 
+loadConfig().then(() => {
+    const { dsn, environment } = global.vectorConfig.sentry || {};
+    if (dsn) {
+        console.log(`Enabling Sentry with dsn=${dsn} environment=${environment}`);
+        Sentry.init({
+            dsn,
+            environment,
+            // We don't actually use this IPC, but we do not want Sentry injecting preloads
+            ipcMode: Sentry.IPCMode.Classic,
+        });
+    }
+});
```

Comment:
could we put this in an `async configureSentry` function, to save doing battle with the promise syntax?